UITests/Global.swift

```diff
@@ -436,11 +370,14 @@ class BrowserUtils {
 
     class func closeLibraryMenu(_ tester: KIFUITestActor) {
         if iPad() {
-            EarlGrey.selectElement(with: grey_accessibilityID("TabToolbar.libraryButton"))
-                .perform(grey_tap())
+            tester.tapView(withAccessibilityIdentifier: "TabToolbar.libraryButton")
         } else {
-            EarlGrey.selectElement(with: grey_accessibilityID("History"))
-                .perform(grey_swipeFastInDirection(GREYDirection.down))
+            // Workaround to be able to swipe the view and close the library panel
+            tester.tapView(withAccessibilityLabel: "Downloads")
+            tester.waitForAnimationsToFinish(withTimeout: 3)
+
+            let view=tester.waitForView(withAccessibilityIdentifier: "Downloads")
+            view?.drag(from: CGPoint(x: 150, y: 80), to: CGPoint(x: 150, y: 530))
```

Comment:
Are these coordinates subject to change?

Reply:
In theory no...I could not find a better way to implement that gesture with KIF, this can be a temporary solution while we investigate a better one.